docs/io/votable/index.rst

```diff
@@ -261,26 +260,40 @@ input.
 
 .. _pedantic-mode:
 
-Pedantic mode
```

Comment:
Very nice find and cleanup, this got missed out in the review cycle of the original PR